web/src/fileview/fileview.js

```diff
@@ -253,8 +341,89 @@ function init(initData) {
         $a.focus();
         window.location = $a.attr('href');
       }
-    }
+    } else if (String.fromCharCode(event.which) == 'N' ||
+               String.fromCharCode(event.which) == 'P' ||
+               String.fromCharCode(event.which) == 'M') {
+      /** 
+       * There is the description of the logic for next/previous and highlighting.
+       *
+       * M workes only with the variables, we cannot highlight group of variables, but we can highlight part of the
+       * variable.
+       *
+       * In the case we cannot highlight the word, we just use simple next/previous logic.
+       *
+       * Otherwise:
+       *   * If we press M for the new word, that creates new highlighting.
```

Comment:
Our of curiosity: where does the `M` keystroke convention come from?

Reply:
Not really, it is pretty close to N and I had that in my Vim config